### PR TITLE
rb86

### PR DESCRIPTION
**Pull request by ryanbaekr**
## Description

Files needed for default and via layouts for the ryanbaekr rb86

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ryanbaekr/rb86/rules.mk [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (drashna) [inline keyboards/ryanbaekr/rb86/rules.mk:11]:
```suggestion
```

File: keyboards/ryanbaekr/rb86/rules.mk [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (drashna) [inline keyboards/ryanbaekr/rb86/rules.mk:30]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth support
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/ryanbaekr/rb86/readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# rb86
+
+A diy sandwich compact TKL keyboard. 
+
+* Keyboard Maintainer: [ryanbaekr](https://github.com/ryanbaekr)
+* Hardware Supported: rb86 with Elite-C
+* Hardware Availability: 2021/05/15
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/ryanbaekr/rb86/readme.md:9]: Could you add instructions on how to jump into the bootloader, to flash new firmware?

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by ryanbaekr** [comment]
@drashna are you able to approve my workflow?

**Review by tzarc — APPROVED** [review]

**Pull request merged**